wiki/Astro-battery-optimisation.md

```diff
@@ -0,0 +1,55 @@
+# Astro: Battery optimisation
+
+Battery optimisation on Android is currently a mess, and the Astro is no exception.
+
+If you want an app to be able to run in the background; there are currently 3 places where you need to enable it. All of them are within the settings app.
+
+## Why
+
+Most apps don't give you any benefit from running in the background, while they make a massive difference to your battery usage. Yet there are a few use-cases that are really valuable. Eg:
+
+* Getting instant messages (eg Whatsapp, Messenger or Telegram) in a timely manner. If they are not allowed to run in the background, messages will often be delayed until you open the app, or it gets periodically run to check in.
+* Location tracking for navigation (eg Google maps, or GPS Essentials). When they stop running, the navigation or route recording just stops.
+* Music plays for a little while after you go to the home screen, or lock the screen.
+
+## DuraSpeed
+
+Go to Settings, and then look for "DuraSpeed" near the bottom of the list:
+
+![A screenshot showing DuraSpeed in settings.](img/duraspeed.jpg)
+
+Find the app that you want to be able to run in the background, and **enable** it.
+
+![A screenshot showing the enabled and disabled apps in DuraSpeed.](img/duraspeedApps.jpg)
+
+## Battery manager
+
+Go to Settings, and then look for "Battery" near the top of the list:
+
+![A screenshot showing where to find the Battery settings.](img/battery.jpg)
+
+Tap "Battery Manager", then "Restricted apps".
+
+You'll get a list. If the app you want to run in the background is in the list; un-tick it:
+
+![A screenshot showing the confirmation dialog box for removing the background restriction on a specific app.](img/removeRestriction.jpg)
+
+Tap "REMOVE" to confirm.
+
+## Battery optimisation
+
+NOTE that this is not simply another view of the "Battery manager".
```

Comment:
'NOTE' should be adjusted to 'NOTE: blah blah blah'.

Reply:
Done :)